### PR TITLE
Update doc

### PR DESCRIPTION
**Pull request by tdelatorre**
This PR update documentation of the main menu.

Requirements:
- [x] checkout to branch task/1179/update-doc
- [x] Generate doc in local

Then:
- [x] Check it
- [x] Merge PR
- [x] close this [task](https://taiga6.kaleidos.net/project/taiganext/task/1179)

### CODE REVIEW

File: importers.adoc [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+:numbered:
+:source-highlighter: pygments
+:pygments-style: friendly
+
+You can configure importers for Jira, Github, Asana and Trello in your own instance.
+
+== Trello
+
+For configure Trello, you have to go to https://trello.com/app-key and obtaing
+your API key and your oauth secret, and then, configure it in your settings/local.py:
```
Comment (yamila-moreno) [inline importers.adoc:12]: files are named with "`" so instead of settings/local.py it's rendered as `settings/local.py`

Comment (yamila-moreno) [reply importers.adoc:12]: The file for taiga6 (for consistency), should be `settings/config.py`

File: Makefile [hunk 2]
```diff
@@ -30,20 +30,20 @@ doc:
 	asciidoctor -T custom-html5 -o dist/index.html index.adoc
 	asciidoctor -T custom-html5 -o dist/setup-production.html setup-production.adoc
 	asciidoctor -T custom-html5 -o dist/setup-development.html setup-development.adoc
```
Comment (yamila-moreno) [inline Makefile:32]: in order of importance, I'd put setup below "how to contribute" section. I think it's better that someone finds first installation and upgrading. Later we'll find a proper spot for "Development" section.

File: index.adoc [hunk 3]
```diff
@@ -139,6 +100,12 @@ _Taiga_ has a complete REST API (the one used by the web app), check the full AP
 documentation link:api.html[here].
 
 
+[[importers]]
+== Importers
+
+_Taiga_ has an importers, check the full documentation link:importers.html[here].
```
Comment (yamila-moreno) [inline index.adoc:106]: check the phrase, not sure what do you mean

File: setup-faqs.adoc [hunk 4]
```diff
@@ -18,7 +18,7 @@ To do this you should update the url value `api` on frontend config file `dist/c
 Also you should update the domain related configuration on the backend settings file:
 `settings/local.py`.
 
-And finally reload the backend config with: `circusctl reload taiga`
+And finally reload the backend service.
```
Comment (yamila-moreno) [inline setup-faqs.adoc:21]: line 29 typo: "set to false"

File: setup-faqs.adoc [hunk 5]
```diff
@@ -44,6 +44,7 @@ PUBLIC_REGISTER_ENABLED = False
 
 If you disable it, only invited users can create accounts in the instance.
 
+If you use docker you also have to change PUBLIC_REGISTER_ENABLED to false in docker-compose.yml
```
Comment (yamila-moreno) [inline setup-faqs.adoc:47]: this is not "also" but "instead"
if you have your installation from source code:
- change the conf.json
- change config.py
if you have docker installation:
- change PUBLIC_REGISTER_ENABLED in both back and front service

File: Makefile (hunk 2 above)
Comment (tdelatorre) [reply Makefile:32]: Done!

File: importers.adoc (hunk 1 above)
Comment (tdelatorre) [reply importers.adoc]: Done!

File: setup-faqs.adoc (hunk 4 above)
Comment (tdelatorre) [reply setup-faqs.adoc:21]: Done!

File: setup-faqs.adoc (hunk 5 above)
Comment (tdelatorre) [reply setup-faqs.adoc]: Done!

File: index.adoc (hunk 3 above)
Comment (tdelatorre) [reply index.adoc]: Done!

**Pull request merged**